ui-tests/src/test/java/io/syndesis/qe/steps/customizations/connectors/ApiClientConnectorsSteps.java

```diff
@@ -38,30 +41,109 @@ public void checkConnectorsListSize(String userName, String listSize) {
     }
 
     //***************************************************************************
-    //******************************* bulk phases ********************************
+    //******************************* bulk steps ********************************
     //***************************************************************************
 
-    @Then("^(\\w+) creates new API connector \"([^\"]*)\"$")
-    public void createNewApiConnector(String user, String connectorName, DataTable properties) throws Throwable {
```

Comment:
Function of this method may be obvious, but it has ~150 rows, you have like 7 local variables which you set to null... Consider creating new class with those variables (like universal connector) and instead of switches inside of switch create methods for specific property creation so only one switch will be in the main method. One method should be visible on one screen for better transparency. This may have low priority as it works as it is, wdyt @dsimansk

Reply:
I'll review it later in https://github.com/syndesisio/syndesis-qe/issues/398